README.md

```diff
@@ -68,46 +69,50 @@ To choose from three configuration settings, install the [`eslint-config-lwc`](h
+| [lwc/no-document-query](./docs/rules/no-document-query.md)                                               | disallow DOM query at the document level                          |         |
+| [lwc/no-attributes-during-construction](./docs/rules/no-attributes-during-construction.md)               | disallow setting attributes during construction                   |         |
+| [lwc/no-disallowed-lwc-imports](./docs/rules/no-disallowed-lwc-imports.md)                               | disallow importing unsupported APIs from the `lwc` package        |         |
+| [lwc/no-leading-uppercase-api-name](./docs/rules/no-leading-uppercase-api-name.md)                       | ensure public property doesn't start with an upper-case character |         |
+| [lwc/no-unexpected-wire-adapter-usages](./docs/rules/no-unexpected-wire-adapter-usages.md)               | enforce wire adapters to be used with `wire` decorator            |         |
+| [lwc/no-unknown-wire-adapters](./docs/rules/no-unknown-wire-adapters.md)                                 | disallow usage of unknown wire adapters                           |         |
+| [lwc/valid-api](./docs/rules/valid-api.md)                                                               | validate `api` decorator usage                                    |         |
+| [lwc/valid-track](./docs/rules/valid-track.md)                                                           | validate `track` decorator usage                                  |         |
+| [lwc/valid-wire](./docs/rules/valid-wire.md)                                                             | validate `wire` decorator usage                                   |         |
+| [lwc/no-restricted-browser-globals-during-ssr](./docs/rules/no-restricted-browser-globals-during-ssr.md) | disallow access to global browser APIs during SSR                 |         |
+| [lwc/no-unsupported-ssr-properties](./docs/rules/no-unsupported-ssr-properties.md)                       | disallow access of unsupported properties in SSR                  |         |
+| [lwc/no-node-env-in-ssr](./docs/rules/no-node-env-in-ssr.md)                                             | disallow usage of process.env.NODE_ENV in SSR                     |         |
+| [lwc/ssr/no-restricted-browser-globals](./docs/rules/ssr/no-restricted-browser-globals.md)               | disallow access to global browser APIs during SSR                 |         |
+| [lwc/ssr/no-unsupported-properties](./docs/rules/ssr/no-unsupported-properties.md)                       | disallow access of unsupported properties in SSR                  |         |
+| [lwc/ssr/no-node-env](./docs/rules/ssr/no-node-env.md)                                                   | disallow usage of process.env.NODE_ENV in SSR                     |         |
+
```

Comment:
```suggestion
```
Otherwise it renders incorrectly:
![it's raw text, not formatted table](https://github.com/user-attachments/assets/645364b0-426b-41c7-8712-e0246f2d4bb0)